tg/support/objectproxy.py

```diff
@@ -16,11 +16,12 @@ def __dir__(self):
         return dir_list
 
     def __getattr__(self, attr):
-        if attr.startswith("__"):
+        if attr.startswith("__") or attr == '_is_coroutine':
```

Comment:
Dunder methods were a general solution, but I'd like to avoid growing a chain of `or` on long term. Can't we just ass `_is_coroutine` as an attribute of TurboGearsObjectProxy? It simply can be `None` to make `iscoroutinefunction` happy: https://github.com/python/cpython/blob/3bc94e678f2961eafc9d665898d73afc6204d314/Lib/asyncio/coroutines.py#L20-L23

Reply:
Yep I think that'd work well